pre-commit/validate_copyright.py

```diff
@@ -0,0 +1,64 @@
+import os
+import pathlib
+import datetime
+import tempfile
+import sys
+
+def get_files(app):
+	files = []
+
+	app_dir = pathlib.Path(__file__).resolve().parent.parent.parent / app
+	if not app_dir.is_dir():
+		return files
+
+	for root, _, filenames in os.walk(app_dir):
+		# exclude directories node_modules, dist
+		if "node_modules" in root or "dist" in root:
+			continue
+
+		for filename in filenames:
+			if filename.endswith((".js", ".ts", ".py", ".md")):
+				if os.path.getsize(os.path.join(root, filename)) > 0:
+					files.append(os.path.join(root, filename))
+	return files
+
+def validate_copyright(files):
+	year = datetime.datetime.now().year
+
+	initial_js_string = "// Copyright (c) "
+	initial_py_string = "# Copyright (c) "
+	initial_md_string = "<!-- Copyright (c) "
+
+	copyright_js_string = f"// Copyright (c) {year}, AgriTheory and contributors\n// For license information, please see license.txt\n\n"
+	copyright_py_string = f"# Copyright (c) {year}, AgriTheory and contributors\n# For license information, please see license.txt\n\n"
+	copyright_md_string = f"<!-- Copyright (c) {year}, AgriTheory and contributors\nFor license information, please see license.txt-->\n\n"
```

Comment:
AgriTheory won't be the only copyright holder. We should collect this value from hooks.py Here's how Frappe does it: https://github.com/frappe/frappe/blob/develop/frappe/modules/utils.py#L272